warp/logging/generateMarkdown.py

```diff
@@ -0,0 +1,93 @@
+import json
+import os
+import sys
+
+
+def human_readable_size(size, decimal_places=3):
+    for unit in ["B", "KiB", "MiB", "GiB", "TiB"]:
+        if size < 1024.0:
+            break
+        size /= 1024.0
+    return f"{size:.{decimal_places}f}{unit}"
+
+
+def sizeOfFile(contractName, file, fileName):
+    jsonPath = os.path.abspath(
+        os.path.join(__file__, "../../../benchmark/", fileName + ".json")
+    )
+    if os.path.exists(jsonPath):
+        functionSteps = json.load(open(jsonPath, "r"))
```

Comment:
There is no guarantee that the file will be closed eventually, the information written to the filesystem and the file descriptor returned to the OS, unless `close` is called on it. It is less terse, but we'll have to use `with` with all `open` functions